resources/tx_curr-library.json

```diff
@@ -1,55 +1,55 @@
-        }
-    ]
-}
\ No newline at end of file
+  "resourceType": "Library",
+  "id": "TX-CURR",
+  "url": "http://fhir.org/plir/Library/TX-CURR",
+  "version": "0.0.1",
+  "name": "TX_CURR",
+  "status": "draft",
+  "experimental": true,
+  "relatedArtifact": [
+    {
+      "type": "depends-on",
+      "resource": "http://hl7.org/fhir/Library/FHIRHelpers|4.0.1"
+    },
+    {
+      "type": "depends-on",
+      "resource": "https://openconceptlab.org/orgs/CIEL/sources/CIEL"
+    }
+  ],
+  "parameter": [
+    {
+      "name": "Measurement Period",
+      "use": "in",
+      "min": 0,
+      "max": "1",
+      "type": "Period"
+    },
+    {
+      "name": "Patient",
+      "use": "out",
+      "min": 0,
+      "max": "1",
+      "type": "Patient"
+    },
+    {
+      "name": "Currently taking ARV",
+      "use": "out",
+      "min": 0,
+      "max": "*",
+      "type": "Observation"
+    },
+    {
+      "name": "initial-population",
+      "use": "out",
+      "min": 0,
+      "max": "1",
+      "type": "boolean"
+    }
+  ],
+  "content": [
+    {
+      "contentType": "text/cql",
+      "data": "bGlicmFyeSBUWF9DVVJSIHZlcnNpb24gJzAuMC4xJwogIAp1c2luZyBGSElSIHZlcnNpb24gJzQu\nMC4xJwogIAppbmNsdWRlIEZISVJIZWxwZXJzIHZlcnNpb24gJzQuMC4xJyBjYWxsZWQgRkhJUkhl\nbHBlcnMKICAKY29kZXN5c3RlbSAiQ0lFTCI6ICdodHRwczovL29wZW5jb25jZXB0bGFiLm9yZy9v\ncmdzL0NJRUwvc291cmNlcy9DSUVMJwogIApjb2RlICJBbnRpIFJldHJvdmlyYWwgVGhlcmFweSI6\nICcxNjAxMTknIGZyb20gIkNJRUwiIGRpc3BsYXkgJ0N1cnJlbnRseSB0YWtpbmcgQVJWJwogCmNv\nZGUgIkNvZGVkIEFuc3dlciI6ICcyMjIyJyBmcm9tICJDSUVMIiBkaXNwbGF5ICdZZXMnCiAKcGFy\nYW1ldGVyICJNZWFzdXJlbWVudCBQZXJpb2QiIEludGVydmFsPERhdGVUaW1lPgogIApjb250ZXh0\nIFBhdGllbnQKICAKZGVmaW5lICJJbml0aWFsIHBvcHVsYXRpb24iOgogICAgZXhpc3RzICJBbnRp\nIFJldHJvdmlyYWwgVGhlcmFweSBBZHVsdHMgQW5kIENoaWxkcmVuIgogIApkZWZpbmUgIkFudGkg\nUmV0cm92aXJhbCBUaGVyYXB5IEFkdWx0cyBBbmQgQ2hpbGRyZW4iOgogICAgKCBbT2JzZXJ2YXRp\nb246ICJBbnRpIFJldHJvdmlyYWwgVGhlcmFweSJdICkgbwogICAgICAgIHdoZXJlIG8udmFsdWUu\nY29kaW5nWzFdPSAiQ29kZWQgQW5zd2VyIgogICAgICAgIG9yIG8udmFsdWUuY29kaW5nWzJdPSAi\nQ29kZWQgQW5zd2VyIgogICAgICAgIHdoZXJlIG8uc3RhdHVzIGluIHsnZmluYWwnLCAnYW1lbmRl\nZCcsICdjb3JyZWN0ZWQnfQogICAgICAgIGFuZCBvLmVmZmVjdGl2ZSBpbiBkYXkgb2YgIk1lYXN1\ncmVtZW50IFBlcmlvZCI="
```

Comment:
@gitcliff ,theres some thing wrong.
i  tried to decode this and it gives wrong value.
can you revise your encoding logic ??